server/models/List.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require("mongoose");
+
+const listSchema = new mongoose.Schema({
```

Comment:
shouldn't be there a way to know who created a list? @izzangalaxybyte

Reply:
no i will add it. good point